### ISSUE TITLE
Consider moving to GitHub Pages

### ISSUE THREAD

**Issue by PotatoCurry** [saranchockan/DebateFlow#1]
With the introduction of [GitHub Pages](https://pages.github.com/), I'd recommend you move your [existing website](https://debateflowsupport.weebly.com/) onto such a site. Just a suggestion. :)

**Comment by saranchockan** [comment]
Oh, that's great! Didn't know that feature existed. Thanks for the help

**Comment by PotatoCurry** [comment]
I could help you set it up if there are any issues. I use the it to host the [Clements Tech website](https://clementstech.club) from [this repo](https://github.com/ClementsTechClub/ClementsTechClub/).

**Comment by saranchockan** [comment]
Yeah, but I need to brush up the project a bit. I will probably resume working on it over the summer. If needed help for the website, I will definitely consult you. Thanks :)
